app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -264,11 +315,56 @@ export default Component.extend({
     );
   },
 
+  _saveFastEditDisabled: propertyEqual(
+    "_fastEditInitalSelection",
+    "_fastEditNewSelection"
+  ),
+
   @action
   insertQuote() {
     this.attrs.selectText().then(() => this._hideButton());
   },
 
+  @action
+  _toggleFastEditForm() {
+    if (this._isFastEditable) {
+      this.toggleProperty("_displayFastEditInput");
+
+      schedule("afterRender", () => {
+        document.querySelector("#fast-edit-input")?.focus();
+      });
+    } else {
+      const postId = this.quoteState?.postId;
+      const postModel = this.topic.postStream.findLoadedPost(postId);
```

Comment:
What happens if `this.quoteState` is `null`?

Reply:
I don't think it can actually be null, the current code doesn't do any check of this kind and I'm not aware of any issue about this. In any case if it could be null it should probably be handled at the level where the function decides to show the quote button body, this ? is probably not needed here

Reply:
I asked because the line above literally check if it can be `null`. It's either that line or this line that do something wrong.

Reply:
yes that's what I said, the `?` is not necessary, I removed it locally :+1: